KlockanAPI/src/KlockanAPI.Presentation/appsettings.Development.json

```diff
@@ -1,14 +1,14 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Host=localhost;Port=5432;Database=klockandb;Username=postgres;Password=postgres"
+    "DefaultConnection": "Host=localhost;Port=5432;Database=klockandb;Username=pierrecampoverde;Password=postgres"
   },
   "Logging": {
     "LogLevel": {
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedOrigins": [ "https://localhost", "http://localhost" ],
+  "AllowedOrigins": [ "https://localhost:4200", "http://localhost:4200" ],
   "KeyCloakJwt": {
     "Authority": "http://localhost:8080/realms/Klockan",
     "AuthorizationUrl": "http://localhost:8080/realms/Klockan/protocol/openid-connect/auth",
```

Comment:
Dont change username on DB conection
Check ports, because i think this change with the latest updates on the repo